includes/entities/class-fs-user.php

```diff
@@ -48,6 +48,19 @@ function __construct( $user = false ) {
 			parent::__construct( $user );
 		}
 
+		/**
+		 * @return void
+		 *
+		 * This function removes the deprecated 'is_beta' property from the serialized data.
+		 * Should clean up the serialized data to avoid PHP 8.2 warning on next execution.
+		 */
+		public function __wakeup() {
```

Comment:
@DanieleAlessandra When the method is public, there's no need to include `public`.